sympy/assumptions/handlers/sets.py

```diff
@@ -20,39 +20,43 @@
 
 from .common import test_closed_group
 from ..predicates.sets import (IntegerPredicate, RationalPredicate,
-    IrrationalPredicate, RealPredicate, ExtendedRealPredicate,
-    HermitianPredicate, ComplexPredicate, ImaginaryPredicate,
-    AntihermitianPredicate, AlgebraicPredicate)
+                               IrrationalPredicate, RealPredicate, ExtendedRealPredicate,
+                               HermitianPredicate, ComplexPredicate, ImaginaryPredicate,
+                               AntihermitianPredicate, AlgebraicPredicate, TranscendentalPredicate)
 
 
 # IntegerPredicate
 
 def _IntegerPredicate_number(expr, assumptions):
     # helper function
-        try:
-            i = int(expr.round())
-            if not (expr - i).equals(0):
-                raise TypeError
-            return True
-        except TypeError:
-            return False
+    try:
+        i = int(expr.round())
+        if not (expr - i).equals(0):
+            raise TypeError
+        return True
+    except TypeError:
+        return False
+
 
-@IntegerPredicate.register_many(int, Integer) # type:ignore
+@IntegerPredicate.register_many(int, Integer)  # type:ignore
```

Comment:
You shouldn't have this random space here

Reply:
Actually, the recommendation is 2 spaces before # on a suffix comment, as I recall.